cmd/update.go

```diff
@@ -138,6 +145,10 @@ func newUpdateCmd() *updateCmd {
 	return root
 }
 
+func printBinaryNotFound(bin string) error {
```

Comment:
this function doesn't print anything actually. Maybe rename it to "pathBinaryNotFoundError ?

Reply:
Yes,  that is true ,  no problem with this.